### PR TITLE
Adds initial feature request form 

### PR DESCRIPTION
**Pull request by brian-glovebox**
- sat with @Los-GB to create the first cut of this form, let me know what you think!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request_form.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      multiple: true
+      options:
+        - PHA-Web
+        - PHA-iOS
+        - PHA-Android
+        - HQ
+        - System
+        - Marketing Website
+    validations:
+      required: true
```
Comment (ARolek) [inline .github/ISSUE_TEMPLATE/feature_request_form.yaml]: I don't think this is needed as the repo the issue originates on will indicate this.

File: .github/ISSUE_TEMPLATE/feature_request_form.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    attributes:
+      label: What Problem are we solving?
+      description: Please provide context to the problem we are solving and how this impacts our stakeholders (i.e. users, clients, etc.)
+    validations:
+      required: true
+  - type: textarea
+    id: user-story
+    attributes:
+      label: Please provide User Story for impacted stakeholders.
+      description: Select from dropdown below.
```
Comment (ARolek) [inline .github/ISSUE_TEMPLATE/feature_request_form.yaml]: What dropdown is this referring to?

Comment (ARolek) [reply .github/ISSUE_TEMPLATE/feature_request_form.yaml]: We could add an example, "As an Agent, when I ______ I want to be able to ______"

**Review by ARolek — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request_form.yaml (hunk 1 above)
Comment (brian-glovebox) [reply .github/ISSUE_TEMPLATE/feature_request_form.yaml]: removed

File: .github/ISSUE_TEMPLATE/feature_request_form.yaml (hunk 2 above)
Comment (brian-glovebox) [reply .github/ISSUE_TEMPLATE/feature_request_form.yaml]: adjusted this to make sense

File: .github/ISSUE_TEMPLATE/feature_request_form.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      required: true
+  - type: textarea
+    id: acceptance-criteria
+    attributes:
+      label: Acceptance Criteria
+      description: Please provide acceptance criteria for this feature request.
+      render: shell
+      validations: 
+      required: true
+    - type: textarea
```
Comment (ARolek) [inline .github/ISSUE_TEMPLATE/feature_request_form.yaml]: I think the indentation is off on this node.

**Review by ARolek — APPROVED** [review]
LGTM pending the indentation fix. Can you please squash all commits into a single commit prior to merging?

**Review by ARolek — APPROVED** [review]

**Pull request merged**